azure-resources/Network/virtualWans/recommendations.yaml

```diff
@@ -0,0 +1,147 @@
+
+- description: Migrate non zone redundant Azure Firewalls to Zone Redundant Azure firewalls by utilizing Firewall Manager
+  aprlGuid: f92e5b09-ca8d-488f-bf91-cf761d261da6
+  recommendationTypeId: null
+  recommendationControl: High Availability
+  recommendationImpact: High
+  recommendationResourceType: Microsoft.Network/azureFirewalls
+  recommendationMetadataState: Active
+  longDescription: Secure Hub might have ended deployed with no Availability Zones. Consider re-deploying your Secure Hub using Firewall Manager so Availability Zones can be configured.
+  potentialBenefits: Reliability of Azure Firewall
+  pgVerified: false
+  publishedToLearn: false
+  publishedToAdvisor: false
+  automationAvailable: no/arg
+  tags: null
+  learnMoreLink:
+    - name: Secure your v-Hub using Azure Firewall Manager
+      url: "https://learn.microsoft.com/en-us/azure/firewall-manager/secure-cloud-network"
+
+- description: Enable routing intent to enable Secured Hubs Interconnectivity
+  aprlGuid: 53cc7cbc-3f6d-4dbe-8018-c1a075002dd0
+  recommendationTypeId: null
+  recommendationControl: High Availability/Business Continuity/
+  recommendationImpact: High
+  recommendationResourceType: Microsoft.Network/virtualHubs/
+  recommendationMetadataState: Active
+  longDescription: |
+    Routing Intent will enable Inter Hub connectivity. This is recommended if you have Secured Hubs and no Custom Route Tables to establish Interconnectivity between the Vnets across v-Hubs.
+  potentialBenefits: Remove assymetrical flow for Spoke to Spoke connectivity.
+  pgVerified: true
+  publishedToLearn: false
+  publishedToAdvisor: false
+  automationAvailable: arg
+  tags: null
+  learnMoreLink:
+    - name: How to configure v-Hub routing intent and routing policies
+      url: "https://learn.microsoft.com/en-us/azure/virtual-wan/how-to-routing-policies"
+
+- description: Enable routing intent to send private and internet traffic via Azure Firewall or Security Solution deployed in the v-Hub.

```

Comment:
Similar comment to the previous one. While this may be a good practice, is it focused on improving resiliency, or aligning to architetural best practices?

Reply:
My train of thought here is, enabling the data path across virtual hubs in case of an outage so it doesn't have to traverse through on premises. This would be recommended if:
- Customer has two secure hubs
- Customer is not using custom Route tables
- Customer doesn't have Routing intent enabled.

Let me know your thoughts

Reply:
My feedback is around line 39 - "Enable routing intent to send private and internet traffic via Azure Firewall or Security Solution deployed in the v-Hub."

Reply:
This is a good point, I wonder if we can collapse both recommendations. Securing traffic is a requirement to enable routing intent, which will enable that inter hub link communication that otherwise wouldn't be enabled if there's firewalls in each hub.

Reply:
I would also say to this:
This is an architectural recommendation, and does not necessarily make VWAN itself more resilient in terms of configuration, so my feeling, based on having this discussion with Rodrigo in the past, is that this guidance would be associated with architecture, not specifically VWAN itself.

Reply:
Removed the recommendations pertaining to routing intent

Reply:
@ehaslett is it ok to resolve this conversation? You happy with this?